src/routes/categories/components/__tests__/CategoryListItem.spec.js

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 import { shallow } from 'enzyme'
 import moment from 'moment'
 
-import CategoryListItem from '../CategoryListItem'
+import CategoryEntry from '../CategoryEntry'
```

Comment:
I think the name of CategoryListItem would be fine here.

Reply:
As I took the City_Entry_ as model I came to the conclusion that it is quite confusing to have both names around. especially as the City_Entry_ also has a CityListItem. But I agree, that if it's about consistency, then I should've also renamed the test case. 😅 
But of course you can also tell me to revert to CategoryListItem if I am allowed to also rename the CityEntry 😄